packages/taquito-sapling/src/taquito-sapling.ts

```diff
@@ -30,15 +30,18 @@ import {
 } from './types';
 import { SaplingTransactionBuilder } from './sapling-tx-builder/sapling-transactions-builder';
 import { DEFAULT_BOUND_DATA, DEFAULT_MEMO } from './constants';
+import { InMemoryProvingKey } from './sapling-keys/in-memory-proving-key';
 
 export { SaplingTransactionViewer } from './sapling-tx-viewer/sapling-transaction-viewer';
 export { InMemoryViewingKey } from './sapling-keys/in-memory-viewing-key';
 export { InMemorySpendingKey } from './sapling-keys/in-memory-spending-key';
+export { InMemoryProvingKey } from './sapling-keys/in-memory-proving-key';
```

Comment:
are these two imports of the same thing?

Reply:
The first one is an import and the second an export. In the main file of the package, we need to explicitly export all classes that we want to make available to the users.